Auth0Tests/CredentialsManagerSpec.swift

```diff
@@ -303,6 +302,17 @@ class CredentialsManagerSpec: QuickSpec {
                 }
             }
             
+            context("custom keychain", closure: {
+                beforeEach {
+                    let storage = A0SimpleKeychain(service: "test_service")
+                    credentialsManager = CredentialsManager(authentication: authentication,
+                                                            storage: storage)
+                }
+                
+                it("should accept custom keychains") {
+                    expect(credentialsManager.storage.service).to(match("test_service"))
```

Comment:
This is looking better, to support restoring the access level to private of the storage property.  You should test against the `storage` instance. So `expect(storage.service` should work fine.